populus/compilation/__init__.py

```diff
@@ -35,7 +35,7 @@ def compile_project_contracts(project):
     compiler_backend = project.get_compiler_backend()
     compiled_contract_data = compiler_backend.get_compiled_contract_data(
         source_file_paths=all_source_paths,
-        import_remappings=None,
+        import_remappings=project.config.get('compilation.import_remappings',None)
```

Comment:
*nitpick*.  This is the same as `project.config.get('compilation.import_remappings')` as the `config.get` method will return `None` if the key is not present.